src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -296,23 +305,33 @@ public void playerInfo(String username) {
       return;
     }
 
-    playerInfoTooltipController.setPlayerInfoBean(playerInfoBean);
+    playerCardTooltipController.setPlayer(playerInfoBean);
 
-    playerInfoTooltip = new Popup();
-    playerInfoTooltip.getContent().setAll(playerInfoTooltipController.getRoot());
-    playerInfoTooltip.setAnchorLocation(PopupWindow.AnchorLocation.CONTENT_BOTTOM_LEFT);
-    playerInfoTooltip.show(getRoot().getTabPane(), lastMouseX, lastMouseY - 10);
+    playerCardTooltip = new Popup();
+    playerCardTooltip.getContent().setAll(playerCardTooltipController.getRoot());
+    playerCardTooltip.setAnchorLocation(PopupWindow.AnchorLocation.CONTENT_BOTTOM_LEFT);
+    playerCardTooltip.show(getRoot().getTabPane(), lastMouseX, lastMouseY - 10);
   }
 
   /**
    * Called from JavaScript when user no longer hovers over a user name.
    */
   public void hidePlayerInfo() {
-    if (playerInfoTooltip == null) {
+    if (playerCardTooltip == null) {
       return;
     }
-    playerInfoTooltip.hide();
-    playerInfoTooltip = null;
+    playerCardTooltip.hide();
+    playerCardTooltip = null;
+  }
+
+  /**
+   * Called from JavaScript when user no longer hovers over a user name.
+   */
+  public void messagePlayerFromChat(String username) {
```

Comment:
Sounds like "send a message to the player from chat", I'd call this `openPrivateMessageTab`